Phases/Assessments/README.md

```diff
@@ -0,0 +1,131 @@
+# Phase Assessments
+
+This document explains how phase assessments will work. To learn what each phase assessment will cover and how to prepare, see the phase-specific assessment pages:
+
+- [Phase 1](./Phase_1.md)
+- [Phase 2](./Phase_2.md)
+- [Phase 3](./Phase_3.md)
+
+Note that there is no assessment for phases 4 or 5.
+
+**Table of Contents**
+
+<!-- TOC depthFrom:2 depthTo:3 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Schedule & Pacing](#schedule-pacing)
+- [The Process](#the-process)
+	- [Requesting an Early Assessment](#requesting-an-early-assessment)
+- [Preparing for the Assessment](#preparing-for-the-assessment)
+- [Getting Support](#getting-support)
+- [Evaluation / Interview](#evaluation-interview)
+	- [Artifact Evaluation](#artifact-evaluation)
+	- [Technical Interview](#technical-interview)
+- [Membership Review](#membership-review)
+
+<!-- /TOC -->
+
+## Schedule & Pacing
+
+There are clear pacing expectations for every learner who joins the guild. Specifically, each phase is designed to contain curriculum that should be able to be completed in 6 weeks, but *must* be completed within 8 weeks. This pacing requirement *includes* the 1 week that will be necessary to go through the assessment process.
```

Comment:
completed _including_ the assessment, so it's really 5 or 7 weeks to complete, followed by a week to assess.